src/core/messenger/passback.ts

```diff
@@ -205,15 +205,34 @@ const init = (register: RegisterListener): void => {
 										`Passback: ad height is ${adHeight}`,
 									);
 									void fastdom.mutate(() => {
-										const slotHeight = `${
-											adHeight + adLabelHeight
-										}px`;
+										//The 100 is to redue CLS if Teads passes back so we don't resize back to 274px for desktop if we get an mpu
+										const slotHeight =
+											getCurrentBreakpoint() === 'mobile'
+												? `${
+														adHeight + adLabelHeight
+												  }px`
+												: `${
+														adHeight +
+														adLabelHeight +
+														100
+												  }px`;
```

Comment:
```suggestion
												: `${
													adSizes.outstreamDesktop.height +
														adLabelHeight
												  }px`;
```

Reply:
Done in 8ab6cacd77fd5b4a342079eab203c4b96b35a692